CRM/Upgrade/Incremental/sql/5.7.alpha1.mysql.tpl

```diff
@@ -13,3 +13,13 @@ UPDATE civicrm_navigation SET icon = "crm-i fa-exchange" WHERE name = "Grants" A
 UPDATE civicrm_navigation SET icon = "crm-i fa-cog" WHERE name = "Administer" AND domain_id = {$domainID} AND icon IS NULL;
 UPDATE civicrm_navigation SET icon = "crm-i fa-life-ring" WHERE name = "Support" AND domain_id = {$domainID} AND icon IS NULL;
 UPDATE civicrm_navigation SET icon = "crm-i fa-bar-chart" WHERE name = "Reports" AND domain_id = {$domainID} AND icon IS NULL;
+
+-- CRM-19751
+INSERT INTO `civicrm_option_group` (`name`, `title`, `data_type`, `is_reserved`, `is_active`, `is_locked`)
```

Comment:
i believe you will need to sql translate title, and the relevant values same with the label column of option_value

Reply:
also @eileenmcnaughton would this be better done via PHP?

Reply:
@twomice @seamuslee001 yes better to add option values using the BAO_OptionGroup::ensureOptionGroupExists function.

However, @totten has rightly been questioning the use of option groups as opposed to specifying a function that returns a hard-coded list. Obviously the difference is flexibility & the question is whether it is 'good flexibility'  or 'bad flexibility'. I'm perhaps inclined to lean towards sticking to the hard-coded list on this - unless we can forsee when someone would want to add additional options and feel comfortable there will no implications / expectations coming off that (not it can be changed later if someone comes up with a use case.

Reply:
Yeah, I think frankly hard-coded options would be better here -- honestly, when are we ever going to want to change this? @eileenmcnaughton do you know of an example off-hand where such a function is being used?